run_all.py

```diff
@@ -61,11 +69,24 @@ def run():
     arguments = sys.argv[1:]
     subprocess.run(["python3", "main.py"] + arguments, check=True)
 
+def print_art(art, scale):
+    if not art:
+        return
+
+    print('')
+    for l in art:
+        for s in range(scale):
+            for c in l:
+                print(c*scale, end='')
+            print('')
+    print('')
+
 
 if __name__ == "__main__":
     (temp_path, driver_repo_path, branch) = setup_environment()
 
     for driver in drivers:
+        print_art(driver.get("art", ""), 2)
```

Comment:
You're already checking if art exists, so you don't need to set a default for art.

Reply:
I find the lines pleasing and the theme thought provoking. The palette is a little monochromatic for my taste though.

Reply:
I'm not sure it is worth the effort to add colours.. As far as I know logs in TeamCity will not render that anyway ;-)